import_logs.py

```diff
@@ -2633,6 +2633,8 @@ def filtered_line(line, reason):
 
                 # in case a format parser included enclosing quotes, remove them so they are not
                 # sent to Matomo
+                if hit.user_agent is None:
+                    hit.user_agent = ''
```

Comment:
```suggestion
                if hit.user_agent is None:
                    hit.user_agent = ''
                # in case a format parser included enclosing quotes, remove them so they are not
                # sent to Matomo
```